package.json

```diff
@@ -161,7 +161,6 @@
     "jszip": "^3.6.0",
     "lodash": "^4.17.21",
     "latest-version": "^5.1.0",
-    "lodash": "^4.17.21",
```

Comment:
Any reason why lodash is removed?

Reply:
I didn't do that, don't know why this is.